notesserver/settings/logger.py

```diff
@@ -69,8 +68,8 @@ def get_logger_config(log_dir='/var/tmp',
             },
             "elasticsearch.trace": {
                 'handlers': handlers,
-                'level': 'DEBUG',
-                'propagate': False,
+                'level': 'WARNING',
+                'propagate': True,
```

Comment:
Why do we need to change the propagation?  What other handlers are in the hierarchy of elasticsearch.trace?